extensions/google-shopping-ads/release-notes/index.md

```diff
@@ -15,6 +15,30 @@ The release notes include:
 
 -   {:.new}New features
 -   {:.fix}Fixes and improvements
+-   {:.bug}Known issues
+
+### v2.0.0
+
+Google Shopping ads Channel 2.0.0 is generally available for versions 2.2.4+ and 2.3.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}.
+
+- {:.fix} **Resolved Configurable Product Issues**: Configurable product (parent/child) handling has been improved.
+    - <!--3372-->Corrected an issue preventing child products from inheriting the parent description and causing Google to disapprove products for missing descriptions
```

Comment:
The numbered issues, are these internal CHAN or MAGEDOC? I would include CHAN before each ticket number. Even for internal. If others need to review, it's helpful to know.

Reply:
yeah, these were originally just for me.   added CHAN- for each of the numbers.

Reply:
Thanks, Ken! We use issue numbers in all release notes for us, devs, review, etc :)